eni.tf

```diff
@@ -0,0 +1,27 @@
+resource "aws_network_interface" "default" {
+  count     = "${var.additional_ips_count}"
+  subnet_id = "${var.subnets[0]}"
```

Comment:
We should, probably, drop `list` for `var.subnets`.
As `list` is not being processed now.

Reply:
https://cloudposse.atlassian.net/browse/CPD-115